lib/compat/wordpress-5.9/script-loader.php

```diff
@@ -63,7 +63,7 @@ function gutenberg_enqueue_global_styles_assets() {
 function gutenberg_enqueue_block_support_styles( $style ) {
 	$action_hook_name = 'wp_footer';
 	if ( wp_is_block_theme() ) {
-		$action_hook_name = 'wp_enqueue_scripts';
```

Comment:
@ndiego Why not add:

```
$action_hook_priority = 999;
```

Then the block support styles can load after the global styles as default priority on hooks is 10. If you are looking to do the reverse, you can make the priority 9 or lower as it will load before the priority of 10. Does this make sense?